redfish-core/include/utils/hw_isolation.hpp

```diff
@@ -143,7 +200,19 @@ inline void
             BMCWEB_LOG_ERROR(
                 "DBus response error [{} : {}] when tried to get the hardware isolation entry for the given resource dbus object path: ",
                 ec.value(), ec.message(), resourceObjPath.str);
-            messages::internalError(asyncResp->res);
+            // The error code (53 == Invalid request descriptor) will be
+            // returned if dbus doesn't contains "isolated_hw_entry" for
+            // the given resource i.e it is not isolated to deisolate.
+            // This case might occur when resource are in the certain state
+            if (ec.value() == 53)
+            {
```

Comment:
So,  we can use `EBADR` here instead of `53`. 
There are many places that bmcweb does the same.
```
if (ec.value() == EBADR)
```

Reply:
Will do. Thanks